Templates/AddTestsToProject/template.json

```
@@ -0,0 +1,42 @@
+{
+    "template_name": "AddTestsToProject",
+    "origin": "https://github.com/o3de/o3de",
+    "license": "Apache-2.0",
+    "display_name": "Add Tests to your Project",
+    "summary": "Adds tests to an existing project.  Point it at your project's root, then open YourProject/Gem/Tests folder for more info",
+    "canonical_tags": [],
+    "user_tags": [
+        "AddTestsToProject"
+    ],
+    "icon_path": "preview.png",
+    "copyFiles": [
+        {
+            "file": "Gem/Tests/custom_google_test.cmake",
+            "isTemplated": false
```

In order to scrub the copyright header block from the tags, these need to be true even if nothing ends up being replaced
```suggestion
            "isTemplated": true
```

---

Ah, didn't know that.  Will upload a change with it.